spec/presenters/default_deadline_date_time_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe DefaultDeadlineDateTime do
+  describe '#initialize' do
+    it 'should be N business days later' do
+      t = DcTimePresenter.parse('2016-08-29 15:00')
+      expect(DcTimePresenter.convert(t)).to be_within(1).of(t)
```

Comment:
I don't think we want to set a specific time, but since there is an hour and minute selector but not a second selector in the form, shouldn't see assume this is always at 0 seconds?